nis/src/main/java/org/nem/nis/dao/retrievers/MosaicSupplyRetriever.java

```diff
@@ -0,0 +1,114 @@
+package org.nem.nis.dao.retrievers;
+
+import org.hibernate.*;
+import org.hibernate.criterion.*;
+import org.nem.core.model.MosaicSupplyType;
+import org.nem.core.model.mosaic.MosaicId;
+import org.nem.core.model.primitive.*;
+import org.nem.nis.dao.*;
+import org.nem.nis.dbmodel.*;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+/**
+ * Class for for retrieving mosaic supplies.
+ */
```

Comment:
I guess spotless needs to be run on the new files?  so it should be a single line.

Reply:
this PR currently only has changes for core, but if rules look ok, i'll update it for the other projects as well